docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+In other words, when developing with the Go SDK, for example, a developer should feel the sense that they're working within the same general development paradigm as when they're working with the TypeScript SDK.
+However, due to the nature of each language, and how they're implemented, each SDK is slightly unique.
+
+Additionally, it's not always the case that a feature and its accessory capability releases across all the SDKs at the same time.
+For example, a feature or an API is released in one SDK, but through developer feedback, it is unnecessary to include in other SDKs.
+
+The following table lists features of the Temporal SDKs.
+This list is meant to provide context and clarification around the SDK feature sets.
+We encourage you to reach out in Slack or our Community forum if further clarification is needed.
+
+### Core development
+
+All SDKs support the fundamental ability to define functions as either Workflows or Activities.
+
+<!-- [Yes](/dev-guide/go/foundations#develop-workflows) -->
+
+| Feature                                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Workflow Definitions](/workflows#workflow-definition)  | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+| [Activity Definitions](/activities#activity-definition) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Workflow Definition is essentially Workflow code, which orchestrates the execution of Activities, persisting the results.
+[Learn more](/workflows#workflow-definition) in the concepts.
+
+An Activity Definition is a normal function or method that defines a single, well-defined action (either short or long running), such as calling another service, transcoding a media file, or sending an email message.
+For more information, see [Activity Definition](/activities#activity-definition) in the concept page.
+
+## Auth (SSL/mTLS)
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Auth (SSL/mTLS)](#auth-ssltls) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Temporal SDKs support authenticating and authorizing client API calls to the Temporal server using SSL and mTLS.
+SSL (Secure Sockets Layer) and its successor TLS (Transport Layer Security) allow encrypted communication between the SDK client and the Temporal server over a network.
+mTLS (mutual TLS) is a variant where both client and server authenticate each other by verifying the provided certificates.
+This ensures secure identification and access control for API calls made from the SDK clients.
+Configuring SSL or mTLS authentication provides network encryption and protects against man-in-the-middle attacks when communicating with the Temporal server.
+
+## Built-in handler list Query
```

Comment:
I don't think this deserves top-billing alongside the other real features. This is an internal thing that will be used by UI and there's no purpose telling users about something they'll never use directly and not supported anywhere yet anyways.